webclient/src/components/LanguageDropdown/LanguageDropdown.css

```diff
@@ -0,0 +1,19 @@
+.LanguageDropdown {
+}
+
+.LanguageDropdown-item {
+  display: flex;
+  align-items: center;
+}
+
+.LanguageDropdown-item__image {
+  width: 1.5em;
```

Comment:
Yeah, it pairs the image ratio with the font size of its label or whatever else. If we tweak the label font-size, the image size scales appropriately.

Reply:
More simply said, it's a technique used when an image's sized should be set in relation to nearby text.

Reply:
Is it safe to say that this is a strategy for icons?

Reply:
I ask because I normally use this tactic for icons, but constrain the height, not the width.  This is a non-issue if the iconography is all in `1:1` aspect ratio and _never_ deviates, but I've yet to see that strategy hold up over time.  Switching this to `height: 1.5em` in that case would alleviate a bit of heartache later on.

I've also used a react component to handle this just to really drive home the consistency part of it all.  A small, atomic component whose only real job is to apply this style isn't a bad option.  Another tactic is to use utility classes such as `.u-icon-image` to handle the same problem.

Reply:
I suppose it depends on your expectations for your images, and the alignment with other items. In our case, i want the text on the right to line up vertically, so i want to make sure the width of the images are always consistent. I'm assuming a near-consistent  height between all the images.